seqr/views/apis/project_api.py

```diff
@@ -305,17 +305,14 @@ def project_mme_submisssions(request, project_guid):
 
 def _add_tag_type_counts(project, project_variant_tags):
     family_tag_type_counts = defaultdict(dict)
-    note_counts_by_family = VariantNote.objects.filter(saved_variants__family__project=project)\
-        .values('saved_variants__family__guid').annotate(count=Count('*'))
-    num_tags = sum(count['count'] for count in note_counts_by_family)
     note_tag_type = {
         'variantTagTypeGuid': 'notes',
         'name': 'Has Notes',
         'category': 'Notes',
         'description': '',
         'color': 'grey',
         'order': 100,
-        'numTags': num_tags,
+        'numTags': SavedVariant.objects.filter(family__project=project, variantnote__isnull=False).distinct().count(),
```

Comment:
How about adding a test to check `numTags` in the response changed?

Reply:
I did. I added test fixture data for an extra note on a variant that already had a note, the test started failing and reporting that there were 2 variants with notes instead of 1, I made this change, and the test started passing again

Reply:
I see.